arm-sql/compositeSql.json

```diff
@@ -0,0 +1,23 @@
+{
+  "info": {
+    "title": "SqlManagementClient",
+    "description": "Composite Swagger for Azure Sql Database Client"
+  },
+  "documents": [
+    //"./2015-05-01/swagger/jobaccount.json",
```

Comment:
#Resolved

---
In reply to: [84755024](https://github.com/Azure/azure-rest-api-specs/pull/622#discussion_r84755024) [](ancestors = 84755024)